hazelcast/src/main/java/com/hazelcast/internal/metrics/StringGauge.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2008-2017, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
```

Comment:
OK, removed StringGauge and implemented render method in the Long and Double gauges as suggested.